README.md

```diff
@@ -1 +1,266 @@
+> cd space-traveler
+> ```
+
+### `Install Depedancy and Run app`
+
+To install linters and other project's dependencies run:
+
+```bash
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+**`npm start`**
+
+> Runs the app in the development mode.\
+> Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+>
+> The page will reload when you make changes.\
+> You may also see any lint errors in the console.
+
+**`npm run build`**
+
+> Builds the app for production to the `build` folder.\
+> It correctly bundles React in production mode and optimizes the build for the best performance.
+> 
+> The build is minified and the filenames include the hashes.\
+> Your app is ready to be deployed!
+>
+> See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+#### Getting Started with Create React App
+> - This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+**`Run Test for Linters`**
+
+> For tracking linter errors locally you need to follow these steps:
+> 
+> 1. For tracking linter errors in HTML files run:
+> ```bash 
+> npx hint .
+> ```
+> 
+> 2. For tracking linter errors in CSS or SASS files run:
+> 
+> ```bash
+> npx stylelint "\*_/_.{css,scss}"
+> ```
+>
+> 3. For tracking linter errors in JavaScript files run:
+> 
+> ```bash
+> npx eslint .
+> ```
+
+**`Test Command`**
+
+> For unit testing of the app run:
+> 
+> ```JS
+> npm test
+> ```
+
+**`Test Screenshoot`**
+
+![image](https://user-images.githubusercontent.com/73167960/235324299-af398388-9b60-446f-8bb9-2cba9b775d46.png)
+
+<!-- ### Deployment -->
+
+<!-- [Space Traveler's Hub]() -->
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors: <a name="authors"></a>
+
+The collaborators of this project are.
+
+👤 **Author 1: Chere Lemma** 
+- GitHub: [Chere Lemma](https://github.com/cherelemma)
+- Twitter: [Chere Lemma](https://twitter.com/Chere21271613)
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> - Enhance the responsivness 
+> - Implement sorting coins
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+[Kindly raise issue here](https://github.com/Microverse-Fullstack-Program/cryptocurrency-market/issues)
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly follow me on GitHub 😊
+
+<p align="right">(<a href="#table-contents">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] In the acknowledgment section of your readme, kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).